### PR TITLE
SC-1986 Added implementation for Object Category Api's

### PR DESCRIPTION
**Pull request by rheafernandes**

### CODE REVIEW

File: taxonomy-api/taxonomy-service/app/controllers/v3/ObjectCategoryController.scala [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package controllers.v3
```
Comment (krgauraw) [inline taxonomy-api/taxonomy-service/app/controllers/v3/ObjectCategoryController.scala:1]: this should be v4

File: taxonomy-api/taxonomy-actors/src/test/resources/application.conf [hunk 2]
```diff
@@ -0,0 +1,506 @@
...
+cassandra.lp.consistency.level=QUORUM
+
+
+
+
+content.nested.fields="badgeAssertions,targets,badgeAssociations"
+
+content.cache.ttl=86400
+content.cache.enable=false
+collection.cache.enable=false
```
Comment (krgauraw) [inline taxonomy-api/taxonomy-actors/src/test/resources/application.conf:454]: why are we adding content related configuration in taxonomy-service? Please remove all content related config which is not required in this service.

**Review by krgauraw — CHANGES_REQUESTED** [review]

File: taxonomy-api/taxonomy-service/conf/routes [hunk 3]
```diff
@@ -15,4 +15,10 @@ POST      /framework/v3/publish/:identifier               controllers.v3.Framewo
 # Category API's
 
 
-# Term API's
\ No newline at end of file
+# Term API's
+
+
+# Object Category API's
```
Comment (krgauraw) [inline taxonomy-api/taxonomy-service/conf/routes:21]: please update the package to v4

File: taxonomy-api/taxonomy-actors/src/test/resources/application.conf (hunk 2 above)
Comment (rheafernandes) [reply taxonomy-api/taxonomy-actors/src/test/resources/application.conf]: Done

File: taxonomy-api/taxonomy-service/app/controllers/v3/ObjectCategoryController.scala (hunk 1 above)
Comment (rheafernandes) [reply taxonomy-api/taxonomy-service/app/controllers/v3/ObjectCategoryController.scala]: done

**Review by maheshkumargangula — CHANGES_REQUESTED** [review]

File: schemas/objectcategory/1.0/schema.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
```
[the remark this answers is not in the document]
Comment (rheafernandes) [reply schemas/objectcategory/1.0/schema.json:1]: Okay

**Review by maheshkumargangula — APPROVED** [review]

**Pull request merged**